test/lib/next-test-utils.js

```diff
@@ -96,6 +96,7 @@ export function getFullUrl(appPortOrUrl, url, hostname) {
     parsedUrl.hash = parsedPathQuery.hash
     parsedUrl.search = parsedPathQuery.search
     parsedUrl.pathname = parsedPathQuery.pathname
+    parsedUrl.hash = parsedPathQuery.hash
```

Comment:
```suggestion
```
This is already handled a couple lines up so we can remove it here it looks like.